src/test-data/ComponentsData.js

```diff
@@ -86,6 +85,53 @@ export const componentDefinitionTestData = {
   },
 };
 
+export const externalComponentDefinitionTestData = {
```

Comment:
This feels really similar to `componentDefinitionTestData`. It might be needed duplication, but I was wondering if there was a better way to do this?

Reply:
The actual data contained in `externalComponentDefinitionTestData` is different from the data contained in `componentDefinitionTestData`. I added more component test data for testing, where I would test against a specific set of data depending on whether there is an external or no external Url. The tests were done in a way to as closely mimic what would happen in the viewers as possible.

Reply:
Yeah and I think this goes back to my question above: is there a reason to use custom `componentDefinitionTestData` when we can just test that the `url` loads and matches `externalComponentDefinitionTestData`? It seems that loading a `url` vs loading `defaulturl` should not be that different. We could test that we are able to load something if no `url` is given but otherwise just provide a `url` and test that it loads properly.

Reply:
Another issue with testing for specific examples is that those examples change a lot. Which would mean breaking tests that might not really be broken.